src/user/views/adsManager/views/advanced/components/review/components/AdSetReview.tsx

```diff
@@ -73,7 +74,27 @@ export function AdSetReview({ adSet, idx, errors }: Props) {
         <CustomListItemText
           primary="Conversion"
           secondary={conversionValue(adSet.conversions)}
-          error={hasErrors ? (adSetError.conversions?.[0] as string) : ""}
+          error={
+            hasErrors && adSetError.conversions ? (
+              <>
+                {adSetError.conversions?.type && (
+                  <>
+                    {adSetError.conversions?.type} <br />
+                  </>
+                )}
+                {adSetError.conversions?.observationWindow && (
+                  <>
+                    {adSetError.conversions?.observationWindow} <br />
+                  </>
+                )}
+                {adSetError.conversions?.urlPattern && (
+                  <>
+                    {adSetError.conversions?.urlPattern} <br />
+                  </>
+                )}
+              </>
+            ) : undefined
+          }
```

Comment:
Yuk. Is there not a better way of doing this?

Reply:
I will move it to a component